UPSTREAM_PROJECTS.yaml

```diff
@@ -130,7 +130,7 @@ projects:
           - tag: v1.2.0
       - name: cluster-api-provider-cloudstack
         versions:
-          - tag: v0.4.9-rc3
+          - commit: cefafbebf46668a36a690e076d25a35f6d24ff3f
```

Comment:
can we use a git tag instead of commit?

Reply:
for this fix we may have to go with commit hash unfortunately